src/components/GlobalPreferences/Notifications/SubscriptionsForm.js

```diff
@@ -265,6 +265,11 @@ export default function SubscriptionsForm({
         `}
       >
         <Button
+          css={`
+            background: ${theme.surfaceInteractive};
+            color: ${theme.surfaceContent};
+            border: 1px solid ${theme.border};
```

Comment:
I think these should already be the defaults for `Button`? https://github.com/aragon/aragon-ui/blob/newstyle/src/components/Button/Button.js#L129

Reply:
This is an override for the disabled state so that it stays white.

Reply:
Oh right, I was just reading the comment about it. We should probably think about adding a loading state into `Button` to support this :+1: